packages/utils/telemetry-utils/src/errorLogging.ts

```diff
@@ -112,21 +118,18 @@ export function normalizeError(
     // We have to construct a new Fluid Error, copying safe properties over
     const { message, stack } = extractLogSafeErrorProperties(error, false /* sanitizeStack */);
     const fluidError: IFluidErrorBase = new SimpleFluidError({
-        errorType: "genericError", // Match Container/Driver generic error type
-        fluidErrorCode: "",
+        errorType: normalizeErrorType,
```

Comment:
This will potentially throw off partners who are looking for `genericError`, since `CreateContainerError` was replaced by `normalizeError`.  What's wrong with using `genericError` here?